openapi/v1/paths/post.yaml

```diff
@@ -77,8 +77,8 @@ paths:
         required: true
         description: The ID of the post to operate on.
         schema:
-          type: integer
-          format: int64
+          type: string # Changed from integer
+          format: uuid  # Changed from int64
```

Comment:
changed from integer to string without asking for our feedback. This breaks the convention of re-using PSQL auto-incremental integer